iac/uni-nav.deploy.yml

```diff
@@ -110,7 +110,7 @@ Resources:
 
                             - !If
                                 - IsProd
-                                - !Ref AWS::NoValue
+                                - https://go.topcoder.com
```

Comment:
We can simply set the value to a single line, indented at the same level as `- !If`:
```
- https://go.topcoder.com
```